### ISSUE TITLE
No index DB found for project

### ISSUE THREAD

**Issue by suniliosdev** [johnno1962/InjectionApp#1]
I am getting this error 
Please select an entity before applying replacement
No index DB found for project:

**Comment by johnno1962** [comment]
Do you have text selected in your project in Xcode?

**Comment by suniliosdev** [comment]
yes I did, attached a screenshot from injection app

![screen shot 2017-01-10 at 9 59 13 am](https://cloud.githubusercontent.com/assets/5010202/21793915/a9e36654-d71b-11e6-881b-6b44d789ef63.png)

**Comment by johnno1962** [comment]
Thanks for getting back. It’s not finding your derived data directory. Do you have a custom one set in your Xcode preferences? Does the directory mentioned in the error exist? I can TeamView if you like.

**Comment by suniliosdev** [comment]
Yes, I custom path for derived data. Is that only reason of error and how could I solve this. ready for TeamView

**Comment by johnno1962** [comment]
I’m afraid injectoin won’t be able to work with a custom path as it has no way of knowing where your build logs are. Relative to project works.

**Comment by suniliosdev** [comment]
I changed path to Default, and got error when trying to inject. please refer the screenshot
![screen shot 2017-01-12 at 2 44 18 pm](https://cloud.githubusercontent.com/assets/5010202/21883579/e49f9ab8-d8d5-11e6-8af1-facd849b63b0.png)

**Comment by johnno1962** [comment]
Looks like you have your Xcode at a non-standard location (with  perhaps a space in the name?) Injection is having trouble running xcodebuild. Can you open up the app bundle and change line 54 of common.pm which is currently:
    $xcodeApp = "/Applications/Xcode.app" if !-d $xcodeApp;
to
    $xcodeApp = "" if !-d $xcodeApp;

**Comment by suniliosdev** [comment]
Ok I made this changes. Now when I try to inject, red badge appeared on inject icon and my simulator stucks.

**Comment by johnno1962** [comment]
I can TeamView if you like, send me your details injectionapp at johnholdsworth.com
